llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -873,16 +926,22 @@ void MemorySanitizer::createUserspaceApi(Module &M,
   // Create the callback.
   // FIXME: this function should have "Cold" calling conv,
   // which is not yet implemented.
+  StringRef WarningFnName = getWarningFnName(
+      TrackOrigins, Recover,
+      ClEmbedFaultingInst != MSanEmbedFaultingInstructionMode::None);
+  SmallVector<Type *, 4> ArgsTy = {};
```

Comment:
Should't this be `SmallVector<Type *, 2>`? I don't see four things being put in

Reply:
I've shrunk it